### ISSUE TITLE
[Documentation] Incorrect command for installing template

### ISSUE THREAD

**Issue by celluj34** [AzureAD/microsoft-identity-web#351]
### Documentation related to component
Web API Template

### Please check all that apply

- [ ] typo
- [ ] documentation doesn't exist
- [ ] documentation needs clarification
- [x] error(s) in the example
- [ ] needs an example

### Description of the issue
The instructions on installing the new preview templates is incorrect. The affected section is located at https://github.com/AzureAD/microsoft-identity-web/wiki/web-api-template#install-the-templates-in-dotnet-core

```
cd bin
cd Debug
dotnet new -i Microsoft.Identity.Web.ProjectTemplates.0.2.0-preview.nupkg
```

should be
```
cd bin
cd Debug
dotnet new -i Microsoft.Identity.Web.ProjectTemplates::0.2.0-preview
```

**Comment by jennyf19** [comment]
@celluj34 what error did you get?

**Comment by celluj34** [comment]
![image](https://user-images.githubusercontent.com/10264021/88205591-a6509d00-cc12-11ea-86a2-d6616f22efc7.png)

**Comment by celluj34** [comment]
The "new" command is what's recommended by nuget itself: https://www.nuget.org/packages/Microsoft.Identity.Web.ProjectTemplates/0.2.0-preview

**Comment by jmprieur** [comment]
@celluj34 : what version of .NET Core do you use? I cannot repro this?

**Comment by celluj34** [comment]
Here's my `dotnet --info`:

![image](https://user-images.githubusercontent.com/10264021/88215126-cedf9380-cc20-11ea-9d2f-e0dc1d6f98d0.png)

**Comment by jennyf19** [comment]
Included in [0.2.2-preview release](https://github.com/AzureAD/microsoft-identity-web/releases/tag/0.2.2-preview)
